src/app/entities/courses/components/course-item/course-item.component.html

```diff
@@ -1,13 +1,16 @@
 <ng-container *ngIf="course">
-  <section class="course-item-container">
+  <section class="course-item-container" [class.course-item-container--top-rated]="course.topRated" [appShowCourseStatus]="course.creationDate">
```

Comment:
for long bindings is is good idea to make then on new lines:
```html
<section 
  class="course-item-container"
  [class.course-item-container--top-rated]="course.topRated"   
  [appShowCourseStatus]="course.creationDate"> 
    ...
```

Reply:
Makes sense